lib/tests/group.rs

```diff
@@ -328,7 +328,7 @@ async fn select_multi_aggregate() -> Result<(), Error> {
 			},
 			{
 				group: 2,
-				one: 7.6,
+				one: 7.6000000000000005,
```

Comment:
A side effect of not promoting to decimals all over the place is that some values exhibit floating-point rounding errors. It is generally not a good practice to compare floats for equality after doing math. I tried `math::fixed(math::some_aggregate(field), 2)` but that didn't seem to work.